crates/re_log_types/src/msg_bundle.rs

```diff
@@ -101,6 +101,34 @@ impl<C> SerializableComponent for C where
 {
 }
 
+/// A [`Component`] that fulfils all the conditions required to be deserialized from an Arrow
+/// payload.
+///
+/// Note that due to the use of HRTBs in `arrow2_convert` traits, you will still need an extra HRTB
```

Comment:
`HRTB` for the uninitiated?

Reply:
https://doc.rust-lang.org/nomicon/hrtb.html

An old-school workaround for the lack of generic associated types... which actually have landed since then, at least in part.
We should be able to make those APIs in `arrow2_convert` friendlier pretty soon.